### DISCUSSION TITLE
有没有TVM支持int8量化的demo

### DISCUSSION THREAD

**Discussion by lizexu123** [mlc-ai/mlc-zh#87] [General]
最近在做tvm有关加速的问题，看到TVM支持Int8量化，想借鉴一下有没有demo

**Comment by Hzfengsy** [comment]
https://tvm.apache.org/docs/how_to/deploy_models/deploy_prequantized.html

**Reply by lizexu123** [reply]
这一篇好像是TVM加载量化后的模型，我试了一个
![image](https://user-images.githubusercontent.com/39205361/182013083-31550715-236b-46bf-a7b8-e97984e311fe.png)
mo'xing'sh模型是resnet18,在cpu上，量化后的效果不如量化前，但是据说用autotvm可以比pytorch的快，正在实施中
